Podfile

```diff
@@ -92,30 +116,38 @@ abstract_target 'defaults' do
     project 'rn/Teacher/ios/Teacher.xcodeproj'
     firebase_pods
   end
-  
+
   target 'TeacherTests' do
     project 'rn/Teacher/ios/Teacher.xcodeproj'
     firebase_pods
   end
-  
+
   target 'Student' do
     project 'Student/Student.xcodeproj'
     firebase_pods
-    pod 'lottie-ios', '~> 3.1.8'
+    lottie
   end
-  
+
   target 'StudentUnitTests' do
     project 'Student/Student.xcodeproj'
     firebase_pods
-    pod 'lottie-ios', '~> 3.1.8'
+    lottie
   end
-  
+
   target 'CanvasCore' do
     project 'CanvasCore/CanvasCore.xcodeproj'
     canvas_crashlytics_rn_firebase_pods
   end
 end
 
+pre_install do |installer|
+  # dSYMs cause problems, will be fixed in cocoapods 1.10
+  # https://github.com/CocoaPods/CocoaPods/pull/9547
```

Comment:
Are these dSYMs important?

Reply:
I'm not sure they are. If we are dealing with a crash within the lib, They would be, but we almost always either work around it or punt.